trackersite/tracker/templates/tracker/ticketack_add.html

```diff
@@ -1,15 +1,24 @@
 {% extends "base.html" %}
-{% load i18n %}
+{% load i18n widget_tweaks %}
 
-{% block title %}{% trans "Confirm" %}: {{ticket.summary}}{% endblock %}
+{% block title %}{% trans "Add" %}: {{ticket.summary}}{% endblock %}
 
 {% block content %}
 <p class="nav"><a href="{% url "ticket_list" %}">{% trans "index" %}</a> &gt; <a href="{% url "ticket_detail" ticket.id %}">{{ticket.summary}}</a> &gt; {% trans "confirmations" %} &gt;</p>
-<h1>{% trans "Confirm" %}: {{ticketack.display}}</h1>
+<h1>{% trans "Add" %}: {{ticketack.display}}</h1>
 
 <form method="POST" action="">
-{{form.as_p}}
-<input type="submit" value="{% trans "Add confirmation" %}">
-<a href="{{ticketack.ticket.get_absolute_url}}">{% trans "back to ticket" %}</a>
-{% csrf_token %}</form></p>
+{% for field in form %}
+{{ field.error }}
+{{ field.label_tag }} 
+<div class="row">
```

Comment:
V těle `{% for %}` začínáš `<div>`, ukončuješ ho až mimo tělo. To podle mě v obecných případech nebude fungovat moc dobře.

Reply:
Opraveno, dík.